docs/cli/cli-guide.md

```diff
@@ -9,6 +9,9 @@ APIs. It's divided up into a number of sections, so you can jump right into
 the examples for an API you're interested in. If you're new to CLI's it should
 provide all the information you need to work more directly with Planet's API's.
 
+!!!tip "We want to hear from you!"
+    This is a beta release. We encourage you to test your workflows rigorously. Based on your feedback, we may roll out additional updates to improve your experience. Please share your feedback with us on our [Planet Developers community channel](https://community.planet.com/developers-55).
```

Comment:
this forum is new to me! I think the 'ask a question' template in GH goes to the general forum. Should I update that to this link? (I imagine the answer is yes)

Reply:
No. PM & PMM review of target resulted in final decision of Planet Developers community channel.

Reply:
Jen is just asking if the link that comes up when you submit an issue should go to developers forum instead of main forum. See https://github.com/planetlabs/planet-client-python/blob/main/.github/ISSUE_TEMPLATE/config.yml

The answer should be yes - we should point people who are in this repo at the developers forum.

Reply:
Added commit to this branch to update the forum links to be the same, see https://github.com/planetlabs/planet-client-python/pull/822/commits/2776a83934f3459a33ebd90a197ff9f30ce52c50